mantidproject/mantid
# Create basic mantidqt package layout

**Description of work**

Implements the base bones of the `mantidqt` package and cmake rules to build it.

There is a single test of the import of the module.

**To test:**

* Code review.
* Checkout and build the code
* `ctest -R mantidqt` should produce a successful result

Refs #20381

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
